sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/SetPrefetchCountTest.java

```diff
@@ -20,6 +20,7 @@
 /**
  * Verifies we can use various prefetch options with {@link EventHubConsumerAsyncClient}.
  */
+@Disabled("Set prefetch tests do not work because they try to send very large number of events at once.")
```

Comment:
Do we plan on fixing this? Or should this test be deleted as this will not be supported at all?
If we plan to fix, then please also file an issue and link it here.